### PR TITLE
Resolves #2017

### LINKED ISSUE #2017: Enabled Accounts
One of the most common needs for Get-AdUser is getting enabled accounts. This ought to be a function in Get-AdUser, like -Enabled, but in lieu of that, the documentation should have an example:

Get-ADUser -LDAPFilter "(!userAccountControl:1.2.840.113556.1.4.803:=2)"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2a4eb77a-834c-0412-8f9f-9c515172237e
* Version Independent ID: 0228a754-2177-71d0-92cc-ae144070947e
* Content: [Get-ADUser (activedirectory)](https://docs.microsoft.com/en-us/powershell/module/activedirectory/get-aduser?view=winserver2012-ps)
* Content Source: [docset/winserver2012-ps/activedirectory/Get-ADUser.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2012-ps/activedirectory/Get-ADUser.md)
* Product: **w10**
* Technology: **powershell**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Add example and incorporate changes from PR 2145.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2017

### CODE REVIEW

File: docset/winserver2012-ps/activedirectory/Get-ADUser.md [hunk 1]
```diff
@@ -100,6 +100,13 @@ PS C:\> Get-ADUser -Filter "Name -eq 'ChewDavid'" -SearchBase "DC=AppNC" -Proper
 
 This command gets the user with the name ChewDavid in the Active Directory Lightweight Directory Services (AD LDS) instance.
 
+### Example 5: Get all enabled user accounts
+```powershell
+C:\PS> Get-ADUser -LDAPFilter '(!userAccountControl:1.2.840.113556.1.4.803:=2)'
+```
+
+This command get all enabled user accounts in AD using an LDAP filter.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADUser.md:108]:
```suggestion
This command gets all enabled user accounts in Active Directory using an LDAP filter.
```

File: docset/winserver2012-ps/activedirectory/Get-ADUser.md [hunk 2]
```diff
@@ -349,13 +356,13 @@ When you run a cmdlet outside of an Active Directory provider drive against an A
 When you run a cmdlet outside of an Active Directory provider drive against an AD LDS target, the default value is the default naming context of the target LDS instance if one has been specified by setting the **msDS-defaultNamingContext** property of the Active Directory directory service agent (DSA) object (**nTDSDSA**) for the AD LDS instance.
 If no default naming context has been specified for the target AD LDS instance, then this parameter has no default value.
 
-When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions will be searched.
-If the value of the *SearchBase* parameter is set to an empty string and you are not connected to a GC port, an error will be thrown.
+When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions is searched.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/activedirectory/Get-ADUser.md:359]:
```suggestion
When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions are searched.
```

File: docset/winserver2012r2-ps/activedirectory/Get-ADUser.md [hunk 3]
```diff
@@ -100,7 +100,14 @@ This command gets all of the properties of the user with the SAM account name Ch
...
 
-This command gets the user with the name ChewDavid in the Active Directory Lightweight Directory Services (AD LDS) instance.
+This command gets the user with the name ChewDavid in the Active Directory Lightweight Directory Services (AD LDS) instance.'
+
+### Example 5: Get all enabled user accounts
+```powershell
+C:\PS> Get-ADUser -LDAPFilter '(!userAccountControl:1.2.840.113556.1.4.803:=2)'
+```
+
+This command get all enabled user accounts in AD using an LDAP filter.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/activedirectory/Get-ADUser.md:110]:
```suggestion
This command gets all enabled user accounts in Active Directory using an LDAP filter.
```

File: docset/winserver2012r2-ps/activedirectory/Get-ADUser.md [hunk 4]
```diff
@@ -351,13 +358,13 @@ When you run a cmdlet outside of an Active Directory provider drive against an A
 When you run a cmdlet outside of an Active Directory provider drive against an AD LDS target, the default value is the default naming context of the target LDS instance if one has been specified by setting the **msDS-defaultNamingContext** property of the Active Directory directory service agent (DSA) object (**nTDSDSA**) for the AD LDS instance.
 If no default naming context has been specified for the target AD LDS instance, then this parameter has no default value.
 
-When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions will be searched.
-If the value of the *SearchBase* parameter is set to an empty string and you are not connected to a GC port, an error will be thrown.
+When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions is searched.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/activedirectory/Get-ADUser.md:361]:
```suggestion
When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions are searched.
```

File: docset/winserver2016-ps/activedirectory/Get-ADUser.md [hunk 5]
```diff
@@ -103,6 +103,13 @@ PS C:\> Get-ADUser -Filter "Name -eq 'ChewDavid'" -SearchBase "DC=AppNC" -Proper
 
 This command gets the user with the name ChewDavid in the Active Directory Lightweight Directory Services (AD LDS) instance.
 
+### Example 5: Get all enabled user accounts
+```powershell
+C:\PS> Get-ADUser -LDAPFilter '(!userAccountControl:1.2.840.113556.1.4.803:=2)'
+```
+
+This command get all enabled user accounts in AD using an LDAP filter.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/activedirectory/Get-ADUser.md:111]:
```suggestion
This command gets all enabled user accounts in Active Directory using an LDAP filter.
```

File: docset/winserver2016-ps/activedirectory/Get-ADUser.md [hunk 6]
```diff
@@ -353,13 +360,13 @@ When you run a cmdlet outside of an Active Directory provider drive against an A
 When you run a cmdlet outside of an Active Directory provider drive against an AD LDS target, the default value is the default naming context of the target LDS instance if one has been specified by setting the **msDS-defaultNamingContext** property of the Active Directory directory service agent (DSA) object (**nTDSDSA**) for the AD LDS instance.
 If no default naming context has been specified for the target AD LDS instance, then this parameter has no default value.
 
-When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions will be searched.
-If the value of the *SearchBase* parameter is set to an empty string and you are not connected to a GC port, an error will be thrown.
+When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions is searched.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/activedirectory/Get-ADUser.md:363]:
```suggestion
When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions are searched.
```

File: docset/winserver2019-ps/activedirectory/Get-ADUser.md [hunk 7]
```diff
@@ -103,6 +103,13 @@ PS C:\> Get-ADUser -Filter "Name -eq 'ChewDavid'" -SearchBase "DC=AppNC" -Proper
 
 This command gets the user with the name ChewDavid in the Active Directory Lightweight Directory Services (AD LDS) instance.
 
+### Example 5: Get all enabled user accounts
+```powershell
+C:\PS> Get-ADUser -LDAPFilter '(!userAccountControl:1.2.840.113556.1.4.803:=2)'
+```
+
+This command get all enabled user accounts in AD using an LDAP filter.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Get-ADUser.md:111]:
```suggestion
This command gets all enabled user accounts in Active Directory using an LDAP filter.
```

File: docset/winserver2019-ps/activedirectory/Get-ADUser.md [hunk 8]
```diff
@@ -353,13 +360,13 @@ When you run a cmdlet outside of an Active Directory provider drive against an A
 When you run a cmdlet outside of an Active Directory provider drive against an AD LDS target, the default value is the default naming context of the target LDS instance if one has been specified by setting the **msDS-defaultNamingContext** property of the Active Directory directory service agent (DSA) object (**nTDSDSA**) for the AD LDS instance.
 If no default naming context has been specified for the target AD LDS instance, then this parameter has no default value.
 
-When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions will be searched.
-If the value of the *SearchBase* parameter is set to an empty string and you are not connected to a GC port, an error will be thrown.
+When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions is searched.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Get-ADUser.md:363]:
```suggestion
When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions are searched.
```

File: docset/winserver2022-ps/activedirectory/Get-ADUser.md [hunk 9]
```diff
@@ -103,6 +103,13 @@ PS C:\> Get-ADUser -Filter "Name -eq 'ChewDavid'" -SearchBase "DC=AppNC" -Proper
 
 This command gets the user with the name ChewDavid in the Active Directory Lightweight Directory Services (AD LDS) instance.
 
+### Example 5: Get all enabled user accounts
+```powershell
+C:\PS> Get-ADUser -LDAPFilter '(!userAccountControl:1.2.840.113556.1.4.803:=2)'
+```
+
+This command get all enabled user accounts in AD using an LDAP filter.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/activedirectory/Get-ADUser.md:111]:
```suggestion
This command gets all enabled user accounts in Active Directory using an LDAP filter.
```

File: docset/winserver2022-ps/activedirectory/Get-ADUser.md [hunk 10]
```diff
@@ -353,13 +360,13 @@ When you run a cmdlet outside of an Active Directory provider drive against an A
 When you run a cmdlet outside of an Active Directory provider drive against an AD LDS target, the default value is the default naming context of the target LDS instance if one has been specified by setting the **msDS-defaultNamingContext** property of the Active Directory directory service agent (DSA) object (**nTDSDSA**) for the AD LDS instance.
 If no default naming context has been specified for the target AD LDS instance, then this parameter has no default value.
 
-When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions will be searched.
-If the value of the *SearchBase* parameter is set to an empty string and you are not connected to a GC port, an error will be thrown.
+When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions is searched.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/activedirectory/Get-ADUser.md:363]:
```suggestion
When the value of the *SearchBase* parameter is set to an empty string and you are connected to a GC port, all partitions are searched.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @velkovb, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
@velkovb Borislav, can I kindly ask to return to one specific commit that resolves the issue and all of the other cosmetic changes in another commit? Thank you

**Pull request merged**